.github/ci-hpc-config.yml

```diff
@@ -0,0 +1,21 @@
+build:
+  modules:
+    - ninja
+  modules_package:
+    # Test without openmpi as this repository should not behave differently 
```

Comment:
Please use the MPI version instead. In fact I would that to be the default. There are tests that do halo exchanges.